demo/date-time-picker-basic-demos.html

```diff
@@ -20,6 +20,56 @@ <h3>Pre-selected value</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Placeholders</h3>
+    <vaadin-demo-snippet id="date-time-picker-basic-demos-placeholders">
+      <template preserve-content>
+        <vaadin-date-time-picker date-placeholder="Pick a date" time-placeholder="Pick a time"></vaadin-date-time-picker>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Disabled and readonly</h3>
+    <vaadin-demo-snippet id="date-time-picker-basic-demos-state">
+      <template preserve-content>
+        <vaadin-date-time-picker disabled label="Disabled input" value="2019-09-16T15:00"></vaadin-date-time-picker>
+        <vaadin-date-time-picker readonly label="Readonly input" value="2019-09-16T15:00"></vaadin-date-time-picker>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Initial position of date picker</h3>
+    <vaadin-demo-snippet id="date-time-picker-basic-demos-initial-position">
+      <template preserve-content>
+        <vaadin-date-time-picker label="Birth date and time" initial-position="1980-01-01"></vaadin-date-time-picker>
+      </template>
+    </vaadin-demo-snippet>
+
+    <!--
+    ! FIXME(haprog):
+    ! Uncomment this demo after i18n has been implemented as part of https://github.com/vaadin/vaadin-date-time-picker/issues/4
+    ! since show-week-numbers doesn't do anything without being able to set i18n.firstDayOfWeek for date picker
+
+    <h3>Date picker with week numbers</h3>
```

Comment:
Should we add this demo in scope of the `i18n` PR?
**Not a blocker**

Reply:
I wanted to add it already since we're adding the property `showWeekNumbers` here, so that we won't forget to add the demo for it in #4. I'll uncomment it soon in the PR for #4 anyway.